components/leaderboard/LeaderboardSort.tsx

```diff
@@ -0,0 +1,52 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { SortDirection } from "../../entities/ISort";
+import styles from "./Leaderboard.module.scss";
+
+export default function LeaderboardSort<LeaderboardSortType>(
+  props: Readonly<{
+    sort: {
+      sort: LeaderboardSortType;
+      sort_direction: SortDirection;
+    };
+    setSort: (sort: {
+      sort: LeaderboardSortType;
+      sort_direction: SortDirection;
+    }) => void;
+    s: LeaderboardSortType;
```

Comment:
this "s" can have better naming. Don't understand right away what it does or why its there